src/mgmt/tests/srv_drpc_tests.c

```diff
@@ -786,13 +786,205 @@ test_drpc_list_pools_success_with_pools(void **state)
 	D_FREE(resp.body.data);
 }
 
+/*
+ * dRPC List Containers setup/teardown
+ */
+
+static int
+drpc_list_cont_setup(void **state)
+{
+	mock_ds_mgmt_pool_list_cont_setup();
+
+	return 0;
+}
+
+static int
+drpc_list_cont_teardown(void **state)
+{
+	mock_ds_mgmt_pool_list_cont_teardown();
+
+	return 0;
+}
+
+/*
+ * dRPC List Containers tests
+ */
+static void
+pack_list_cont_req(Drpc__Call *call, Mgmt__ListContReq *req)
+{
+	size_t	len;
+	uint8_t	*body;
+
+	len = mgmt__list_cont_req__get_packed_size(req);
+	D_ALLOC(body, len);
+	assert_non_null(body);
+
+	mgmt__list_cont_req__pack(req, body);
+
+	call->body.data = body;
+	call->body.len = len;
+}
+
+static void
+setup_list_cont_drpc_call(Drpc__Call *call, char *uuid)
+{
+	Mgmt__ListContReq lc_req = MGMT__LIST_CONT_REQ__INIT;
+
+	lc_req.uuid = uuid;	/* pool UUID */
+	pack_list_cont_req(call, &lc_req);
+}
+
+static void
+expect_drpc_list_cont_resp_with_error(Drpc__Response *resp,
+				      int expected_err)
+{
+	Mgmt__ListContResp *lc_resp = NULL;
+
+	assert_int_equal(resp->status, DRPC__STATUS__SUCCESS);
+	assert_non_null(resp->body.data);
+
+	lc_resp = mgmt__list_cont_resp__unpack(NULL, resp->body.len,
+					      resp->body.data);
+	assert_non_null(lc_resp);
+
+	assert_int_equal(lc_resp->status, expected_err);
+
+	mgmt__list_cont_resp__free_unpacked(lc_resp, NULL);
+}
+static void
+expect_drpc_list_cont_resp_with_containers(Drpc__Response *resp,
+					   struct daos_pool_cont_info *cont,
+					   uint64_t cont_len)
```

Comment:
Just a small naming suggestion - maybe exp_cont, exp_len, so it's a little easier to follow expectations while reading below.

Reply:
I missed this when updating the patch. I had modeled it from a pool list function that is similar. However I see the inconsistency with some of the acl functions in the file. If you think it's important to unify the parameter naming of all of the functions I can do that in an updated patch.

static void
expect_drpc_list_pools_resp_with_pools(Drpc__Response *resp,
				       struct mgmt_list_pools_one *pools,
				       size_t pools_len)

Reply:
changing the function for list pools and this one to use exp_ prefix on the arguments.